src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -181,7 +207,7 @@ private static void StopBridge(CommandLineArguments commandLineArgs)
                 Console.WriteLine("A problem was encountered stopping the Bridge:{0}{1}",
                                     Environment.NewLine, problem);
                 Console.WriteLine("Forcing local resource cleanup...");
-                BridgeController.StopBridgeProcess(1);
+                BridgeController.StopBridgeProcess(Bridge_ExitCode_Stopped);
```

Comment:
As soon as we added more exit code than we actually need, we will be confused what exit code to use at each case. So as I recommended earlier, we should just stick with success or failure for most commands (except -require for now).

For the case of this line, we should exit with failure and the line below should be success. They both exit with the same code now.